mappings/net/minecraft/client/gui/hud/in_game/InGameHud.mapping

```diff
@@ -161,5 +231,18 @@ CLASS net/minecraft/unmapped/C_nepwsmgw net/minecraft/client/gui/hud/in_game/InG
 	METHOD m_znsrivym setRecordPlayingOverlay (Lnet/minecraft/unmapped/C_rdaqiwdt;)V
 		ARG 1 description
 	CLASS C_mdshohhj HeartType
+		FIELD f_chhxrkvp hardcoreHalfBlinking Lnet/minecraft/unmapped/C_ncpywfca;
+		FIELD f_ghyqxqvl hardcoreHalf Lnet/minecraft/unmapped/C_ncpywfca;
+		FIELD f_kedkcssv full Lnet/minecraft/unmapped/C_ncpywfca;
+		FIELD f_mapfxoof halfBlinking Lnet/minecraft/unmapped/C_ncpywfca;
+		FIELD f_mlgfossm half Lnet/minecraft/unmapped/C_ncpywfca;
+		FIELD f_rphpsjvq hardcoreFullBlinking Lnet/minecraft/unmapped/C_ncpywfca;
+		FIELD f_slrltipg hardcoreFull Lnet/minecraft/unmapped/C_ncpywfca;
+		FIELD f_wcxevweg fullBlinking Lnet/minecraft/unmapped/C_ncpywfca;
+		METHOD <init> (Ljava/lang/String;ILnet/minecraft/unmapped/C_ncpywfca;Lnet/minecraft/unmapped/C_ncpywfca;Lnet/minecraft/unmapped/C_ncpywfca;Lnet/minecraft/unmapped/C_ncpywfca;Lnet/minecraft/unmapped/C_ncpywfca;Lnet/minecraft/unmapped/C_ncpywfca;Lnet/minecraft/unmapped/C_ncpywfca;Lnet/minecraft/unmapped/C_ncpywfca;)V
```

Comment:
Empty method

Reply:
I'm not entirely sure, do you mean the constructor? It's not empty, it just only assigns the fields to the parameters passed.

Reply:
I meant the mapping, it doesn't have a name or parameters

Reply:
the parameter names were marked as suggestions (which I'm not entirely sure when they are put into the mappings file and when they aren't). I'm relatively certain that it's rather normal for a constructor to not have a mapped name.